### PR TITLE
Add llvm 11.1 to the list of supported versions

### PR DESCRIPTION
**Pull request by Sija**
This allows building compiler again under macOS + brew.

### CODE REVIEW

File: src/llvm/ext/llvm-versions.txt [hunk 1]
```diff
@@ -1 +1 @@
-11.0 10.0 9.0 8.0 7.1 6.0 5.0 4.0 3.9 3.8
+11.1 11.0 10.0 9.0 8.0 7.1 6.0 5.0 4.0 3.9 3.8
```
Comment (asterite) [inline src/llvm/ext/llvm-versions.txt:1]: In my case it's `11.1.0`

Comment (Sija) [reply src/llvm/ext/llvm-versions.txt:1]: Yeah, but it works by omitting the _patch_ part anyway - all the other versions are defined like that.

**Review by straight-shoota — APPROVED** [review]

**Review by sdogruyol — APPROVED** [review]
Yes, please! Thank you @Sija 🙏

**Comment by bcardiff** [comment]
Note that this does not change the current issues with llvm 11. For this kind of changes is totally reasonable to apply patches on homebrew distribution IMO.

**Pull request merged**